lib/optimizer/src/Transformer/ServerSideRendering.php

```diff
@@ -46,6 +56,72 @@ final class ServerSideRendering implements Transformer
         Layout::INTRINSIC,
     ];
 
+    /**
+     * List of elements to exclude from rendering their layout at the server.
+     *
+     * @var string[]
+     */
+    const EXCLUDED_ELEMENTS = [
+        'amp-audio',
+    ];
+
+    /**
+     * Map of attribute names to extraction methods.
+     *
+     * @var string[]
+     */
+    const EXTRACTION_METHOD_MAP = [
+        Attribute::SIZES   => 'extractSizesAttributeCss',
+        Attribute::HEIGHTS => 'extractHeightsAttributeCss',
+        Attribute::MEDIA   => 'extractMediaAttributeCss',
+    ];
+
+    /**
+     * XPath query to retrieve the <style amp-custom> tag, relative to the <head> node.
+     *
+     * @var string
+     */
+    const STYLE_AMP_CUSTOM_XPATH = './/style[@amp-custom]';
+
+    /**
+     * Regex pattern to match a CSS Dimension with an associated media condition.
+     *
+     * @var string
+     */
+    const CSS_DIMENSION_WITH_MEDIA_CONDITION_REGEX_PATTERN = '/(?<media_condition>.*)\s(?<dimension>.*)/m';
+
+    /**
+     * Characters to use for trimming CSS values.
+     *
+     * @var string
+     */
+    const CSS_TRIM_CHARACTERS = " \t\n\r\0\x0B;";
+
+    /**
+     * Maximum size of the CSS styles in bytes.
+     *
+     * @todo Max size is hard-coded for now until we ported over the generated spec into a reusable package.
+     *
+     * @var int
+     */
+    const MAX_CSS_BYTE_COUNT = 75000;
+
+    /**
+     * The <style amp-custom> element that custom CSS styles need to be added to.
+     *
+     * @var DOMElement
+     */
+    private $ampCustomStyleElement;
+
+    /**
+     * Count of bytes to calculate against the AMP size limit for the custom CSS styling.
+     *
+     * AMP only allows for 75.000 bytes of CSS across <style amp-custom> and inline style attributes.
```

Comment:
```suggestion
     * AMP only allows for 75,000 bytes of CSS across <style amp-custom> and inline style attributes.
```

Reply:
As that is a regional thing and both versions read like 75 bytes with three zeros of decimal precision in one half of the world, shouldn't we go with a more neutral `75000` instead?

Reply:
Sounds good 👍